solidity/dashboard/src/forms/common-validators.js

```diff
@@ -22,7 +22,12 @@ export const validateEthAddress = (address, required = true) => {
   }
 }
 
-export const validateAmountInRange = (value, maxValue, minValue = 0) => {
+export const validateAmountInRange = (
+  value,
+  maxValue,
+  minValue = 0,
+  token = KEEP
+) => {
```

Comment:
There is a problem with validation for depositing KEEP in KEEP-only pool.

![image](https://user-images.githubusercontent.com/40306834/115864265-ee694e00-a436-11eb-8404-0e777a93e60b.png)

When you write "0" it shows the message "The value should be greater or equal than <1" (it should just be "1" instead of "<1")

I think the problem lies there:

https://github.com/keep-network/keep-core/blob/d2e9cbc3ceed0610a857d1daae594277b738563a/solidity/dashboard/src/components/KeepOnlyPool.jsx#L297

We pass 1 to the validateAmountInRange but it's 1 wei instead of 1 token. That's why it fails on this if:
https://github.com/keep-network/keep-core/blob/d2e9cbc3ceed0610a857d1daae594277b738563a/solidity/dashboard/src/utils/token.utils.js#L164

It's because `MIN_AMOUNT_TO_DISPLAY` is 1 token (10^18) not 1 wei.

Reply:
Also let's check another places where we use 'validateAmountInRange' because i think same thing is happening in `CreateTokenGrantForm.jsx`